documentation/book/ref-configuring-container-images.adoc

```diff
@@ -17,11 +17,29 @@ Container image which should be used for given components can be specified using
 * `KafkaConnect.spec`
```

Comment:
I see that this file is not included in the main assembly or at least it's never referenced from any other adoc file.

Reply:
This module was already existing. I've simply updated it for how things work now.